packages/paste-website/src/pages/components/progress-steps/index.mdx

```diff
@@ -182,6 +192,42 @@ Use vertical Progress Steps when you’re constrained on vertical space in your
   {verticalExample}
 </LivePreview>
 
+#### Additional content
+
+The vertical, non-interactive Progress Steps component has an optional child component that allows you to add additional content.
+
+- Use it when all steps can be completed on the same screen, either within the step itself or on another surface (ex: Modal).
+- Use it when the process links to other pages and has a unique URL. but a list of all steps need to be in the same screen.
+- Use it when you need to add additional context to each step.
```

Comment:
```suggestion
Use `ProgressStepContent` when:
- All steps can be completed on the same screen, either within the step itself or on another surface (ex: Modal).
- The process links to other pages and has a unique URL. but a list of all steps need to be in the same screen.
- You need to add additional context to each step.
```

Reply:
I updated these to:

Use additional content when each step requires minimal content or actions—like opening a modal, redirecting, or entering text, and users need to complete steps in a specific sequence, with progress visibly tracked throughout.